frontend/src/modules/Survey/Components/Survey.tsx

```diff
@@ -43,13 +59,17 @@ export const Survey = () => {
   // last step's Next button handles sending data
   function finalNext() {
     const mcData = Object.fromEntries(
-      questions.map((question, index) => [question.questionId, answers[index]])
+      questions.question.map((question, index) => [
+        question.question.hash,
+        answers[index],
+      ])
     )
     const surveyData: SurveyData = {
       fullName: nameAnswer,
-      studentId: emailAnswer,
+      studentId: emailAnswer, // why does this keep changing bruh
```

Comment:
as in, why does this keep flipflopping between email and studentId

Reply:
lol let's settle with email shall we?

Reply:
wait so why is it still "studentId" here?

Reply:
oh okay its outdated.